tests/numerics/smoothness_estimator_03.output

```diff
@@ -0,0 +1,3 @@
+
+DEAL::Ok
+DEAL::Ok
```

Comment:
Judging from the source code, this should be in capitals. Did you run this test yet and checked if it passes?

```suggestion
DEAL::OK
DEAL::OK
```

Reply:
Sorry. The source code was modified unintentionally.